tests/jeos/image_info.pm

```diff
@@ -19,22 +19,97 @@ use version_utils qw(is_sle is_opensuse is_openstack);
 use mmapi qw(get_current_job_id);
 use Cwd;
 
-sub ls {
-    my $output = `ls -l`;
-    record_info('ls', $output);
+our $image;
+our $job_url;
+our $db_log = "/tmp/influxdb.txt";
+our $influxdb_server = get_var('INFLUXDB_SERVER');
```

Comment:
Yes, looks like I can simply use `my`, as this `.pm` is not a package. I thought I needed global variables so that they can be accessed by all methods and be able to change them. Thanks!